Library/Formula/contacts-phonetic.rb

```diff
@@ -0,0 +1,17 @@
+class ContactsPhonetic < Formula
+  version "0.2"
+  desc "Add phonetic names to contacts."
+  homepage "https://github.com/Elethom/ContactsUtil-PhoneticName"
+  url "https://github.com/Elethom/ContactsUtil-PhoneticName/archive/#{version}.tar.gz"
+  sha256 "c6a5deccc7e8f20f89d96f9fde86aa4ae04dba5d7492dec680b839e2fbdfb247"
+
```

Comment:
I'm not so sure. Just added it anyways. Don't really have a Mac with lower versions OS X to test it out. Sorry. > <